apps/mobile/src/services/notifications/FCMService.ts

```diff
@@ -0,0 +1,67 @@
+import messaging, { FirebaseMessagingTypes } from '@react-native-firebase/messaging'
+import Logger from '@/src/utils/logger'
+import NotificationsService from './NotificationService'
+import { ChannelId } from '@/src/utils/notifications'
+import { store } from '@/src/store'
+import { savePushToken } from '@/src/store/notificationsSlice'
+
+type UnsubscribeFunc = () => void
+
+class FCMService {
+  getFCMToken = async (): Promise<string | undefined> => {
```

Comment:
I already raised it in another PR: can we please stick to the regular class methods?
```suggestion
  async getFCMToken(): Promise<string | undefined> {
```

Reply:
I meant everywhere 😄 
It should be the default syntax. We only use bound methods when actually needed.

Reply:
https://github.com/safe-global/safe-wallet-monorepo/pull/4787/files#diff-7e343872bff4d38b4057d2b9425b6f1c899a42090e1a59dd213fda8a6f65da30R31

https://github.com/safe-global/safe-wallet-monorepo/pull/4787/files#diff-7e343872bff4d38b4057d2b9425b6f1c899a42090e1a59dd213fda8a6f65da30R42